src/dialogs/more-info/controls/more-info-update.ts

```diff
@@ -87,6 +88,29 @@ class MoreInfoUpdate extends LitElement {
     return ["home_assistant", "home_assistant_os"].includes(type);
   }
 
+  private _isVersionSkipped(): boolean {
+    if (!this.stateObj) {
+      return false;
+    }
+
+    return !!(
+      this.stateObj.attributes.latest_version &&
+      this.stateObj.attributes.skipped_version ===
+        this.stateObj.attributes.latest_version
+    );
+  }
+
+  private _isUpdateButtonDisabled(): boolean {
```

Comment:
Move these functions to https://github.com/dcapslock/ha-frontend/blob/dev/src/data/update.ts#L55